src/ray/util/logging.cc

```diff
@@ -450,25 +349,13 @@ void WriteFailureMessage(const char *data, int size) {
 #endif
 }
 
-bool RayLog::IsFailureSignalHandlerEnabled() {
-  return is_failure_signal_handler_installed_;
-}
-
 void RayLog::InstallFailureSignalHandler() {
 #ifdef _WIN32
   // If process fails to initialize, don't display an error window.
   SetErrorMode(GetErrorMode() | SEM_FAILCRITICALERRORS);
   // If process crashes, don't display an error window.
   SetErrorMode(GetErrorMode() | SEM_NOGPFAULTERRORBOX);
 #endif
-#if defined(RAY_USE_GLOG) || defined(RAY_USE_SPDLOG)
-  if (is_failure_signal_handler_installed_) {
-    return;
-  }
-  google::InstallFailureSignalHandler();
-  google::InstallFailureWriter(&WriteFailureMessage);
```

Comment:
If we remove this, who will call `WriteFailureMessage`? I see that you removed all the signal handler-related code, but if we don't install the signal handler, how would we print stack trace on receiving a signal?

Reply:
Maybe it should not be removed, i will test signal handler.

Reply:
We need to rewrite singalhandler if these functions of glog should be removed.

Reply:
Right, i will deal with signal handler later.